src/crs_linter/linter.py

```diff
@@ -955,3 +958,37 @@ def check_capture_action(self):
                         captured_var_chain_level = 0
                         use_captured_var = False
                         ruleid = 0
+
+    def find_ids_without_tests(self, test_cases, exclusion_list):
+        """
+            s: the parsed structure
+            test_cases: all available test cases
+            exclusion_list_name: file which contains rule id's without tests
+        """
+        rids = {}
+        for i in self.data:
+            # only SecRule counts
+            if i['type'] == "SecRule":
+                for a in i['actions']:
+                    # find the `id` action
+                    if a['act_name'] == "id":
+                        # get the argument of the action
+                        rid = int(a['act_arg']) # int
+                        srid = a['act_arg']     # string
+                        if (rid%1000) >= 100:   # skip the PL control rules
+                            # also skip these hardcoded rules
+                            need_check = True
+                            for excl in exclusion_list:
+                                if srid[:len(excl)] == excl:
```

Comment:
When would `srid` be longer than `excl` and still be equal to `excl`?

Reply:
This makes suitable the script to add exclusion as a "prefix", see [these](https://github.com/airween/crs-linter/blob/2414911fc48f475c6d93109c468fc931effa523c/TESTS_EXCLUSIONS#L1-L6) lines.

```
900
901
905
910
912
949
```
Here we don't need to list all unique rules, it's enough to put the "prefix". Eg. 900XXX rules don't need tests, and so on...